satpy/etc/readers/gld360_ualf2.yaml

```diff
@@ -0,0 +1,157 @@
+reader:
+  description: Gld360Ualf2 reader
```

Comment:
please add a bit more "meat" on the description.

Reply:
I agree, this could describe what Gld360Ualf2 is.